wiki/Beatmap/Genre_and_language/de.md

```diff
@@ -0,0 +1,67 @@
+
+[Beatmaps](/wiki/Beatmap), die auf die Webseite hochgeladen werden, haben Felder für das **Genre** und die **Sprache**, um bei der Kategorisierung von Songs zu helfen. Sie werden als Teil der [Metadaten](/wiki/Client/Beatmap_editor/Song_setup#song-and-map-metadata) einer Beatmap angesehen.
+
+Das Genre und die Sprache haben keine Funktion im Spiel, sind aber als Suchoptionen in [osu!(lazer)](/wiki/Client/Release_stream/Lazer) verfügbar.
+
+## Beatmap-Auflistung
+
+![Die verfügbaren Filter in der Beatmap-Auflistung](img/beatmap-listing-DE.png)
+
+Die [Beatmap-Auflistung](https://osu.ppy.sh/beatmapsets) hat Filter für das Genre und die Sprache im Dropdown-Menü `Mehr Suchoptionen`.
+
+## Das Genre und die Sprache einer Beatmap ändern
+
+Das Genre und die Sprache kann auf der Webseite geändert werden, indem man mit dem Mauszeiger über die Details einer Beatmap fährt und dabei auf das Bleistiftsymbol klickt. Dabei bestimmen die [Benutzergruppe](/wiki/People/User_group) des Nutzers und die [Kategorie](/wiki/Beatmap/Category#aktuelle-kategorien) der Beatmap die jeweiligen Rechte:
+
+- Beatmap-Eigentümer dürfen das, wenn sich die Beatmap in der Kategorie [WIP](/wiki/Beatmap/Category#work-in-progress-und-ausstehend), [Ausstehend](/wiki/Beatmap/Category#work-in-progress-und-ausstehend) oder [Friedhof](/wiki/Beatmap/Category#friedhof) befindet und keine [Nominierungen](/wiki/Beatmap_ranking_procedure#nominations) hat.
+- [Beatmap Nominator](/wiki/People/Beatmap_Nominators) dürfen das, wenn sich die Beatmap in der Kategorie [Qualifiziert](/wiki/Beatmap/Category#qualifiziert), WIP oder Ausstehend befindet.
+- Mitglieder des [Project Loved Teams](/wiki/People/Project_Loved_Team) dürfen das, wenn sich die Beatmap in der Kategorie [Loved](/wiki/Beatmap/Category#loved) befindet.
+- Mitglieder des [Nomination Assessment Teams](/wiki/People/Nomination_Assessment_Team) und des [Globalen Moderationsteams](/wiki/People/Global_Moderation_Team) dürfen das für alle Kategorien.
+
+Eine Beatmap kann nicht [nominiert](/wiki/Beatmap_ranking_procedure#nominations) werden, wenn die Sprache oder das Genre auf `Nicht angegeben` eingestellt ist.
+
+## Liste der Genres
+
+| Genre | Merkmale entsprechender Songs |
+| :-- | :-- |
+| Nicht angegeben | Keine Songs treffen zu. Dieses Genre ist ein Platzhalter, bis ein geeignetes Genre festgelegt ist. |
+| Videospiel | Gemacht für oder populär geworden durch ein Videospiel. Dies wird auch für Arrangements und Remixes dieser Songs verwendet. |
+| Anime | Produziert für oder bekannt geworden durch einen Anime oder ein vergleichbares Medium. |
+| Rock | Mit einem Schwerpunkt auf Gitarren, Schlagzeug und Bass, typischerweise auf der Grundlage von einfachen synkopischen Rhythmen basierend und von Live-Auftritten und einem Fokus auf ernste sowie progressive Motive charakterisiert ([*Rockmusik* auf Wikipedia](https://de.wikipedia.org/wiki/Rockmusik)). |
```

Comment:
Keep in mind that the citation used here is incorrect. It says "unsyncopated", but in the Wikipedia article, it says "syncopated".

Reply:
Closing this note due to #10117 already being merged.